src/Momento.Sdk/Auth/ICredentialProvider.cs

```diff
@@ -0,0 +1,10 @@
+namespace Momento.Sdk.Auth;
+
+using System.Collections.Generic;
+
+public interface ICredentialProvider
+{
+    public string AuthToken { get; }
+    public string ControlEndpoint {get; }
+    public string CacheEndpoint { get; }
```

Comment:
public is already the default visibility here on an interface.

Reply:
Nice, thanks.